example/app.js

```diff
@@ -7,60 +7,55 @@
  */
 require('isomorphic-fetch');
 
-const { resolve } = require('path');
+const {resolve} = require('path');
 const hapi = require('../init'); // hapi = require('cnn-hapi'),
 const healthChecks = require('cnn-health')(resolve(__dirname, './config/healthcheck')).asArray();
 
-
 const server = hapi({
-    basePath: __dirname,
+  basePath: __dirname,
 
-    description: 'A Test Harness for building CNN-HAPI',
-    name: 'testHarness',
-    surrogateCacheControl: 'max-age=60, stale-while-revalidate=10, stale-if-error=6400',
+  description: 'A Test Harness for building CNN-HAPI',
+  name: 'testHarness',
+  surrogateCacheControl: 'max-age=66, stale-while-revalidate=10, stale-if-error=6400',
 
-    loaderIoValidationKey: process.env.LOADER_IO_VALIDATION,
-    port: process.env.PORT,
+  loaderIoValidationKey: process.env.LOADER_IO_VALIDATION,
+  port: process.env.PORT,
 
-    withGoodConsole: true,
-    withSwagger: true,
+  withGoodConsole: true,
+  withSwagger: true,
+  host: 'localhost',
 
-    healthChecks: [...healthChecks, require('./config/otherchecks')],
+  healthChecks: [...healthChecks, require('./config/otherchecks')],
 
-    customHeaders: [{
-        name: 'Connection',
-        value: 'close'
-    }],
-    localTLS: {
-        cert: '.local/localhost.cnn.io.crt',
-        key: '.local/localhost.cnn.io.key',
-        port: process.env.LOCAL_TLS_PORT
-    },
-    metrics: {
-        provider: require('cnn-metrics'),
-        options: {flushEvery: (6 * 1000)}
+  customHeaders: [
+    {
+      name: 'Connection',
+      value: 'close'
     }
+  ],
+  localTLS: {
```

Comment:
highly debated topic, and not worth changing, but I want to arm you with the info. 

According to the standard, which is only issued by Microsoft on this specific case, when using camelCase, only 2 letter acronyms should be capitalized. ie, this should be `localTls`

ref: https://stackoverflow.com/questions/15526107/acronyms-in-camelcase